src/constant_time.rs

```diff
@@ -33,5 +33,5 @@ pub fn verify_slices_are_equal(a: &[u8], b: &[u8])
 }
 
 extern {
-    fn GFp_memcmp(a: *const u8, b: *const u8, len: c::size_t) -> c::int;
+    pub fn GFp_memcmp(a: *const u8, b: *const u8, len: c::size_t) -> c::int;
```

Comment:
Oh, you no doubt meant the direct usage of `GFp_memcmp` more generally, not just this specific change. Agreed.